src/main/java/uk/gov/ons/entities/HybridAddress.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.ons.entities;
+
+import org.springframework.data.annotation.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import uk.gov.ons.util.CreateAddressConstants.CountryCode;
+
+@AllArgsConstructor
+public @Data class HybridAddress {
+
+	@Id
+	private Long uprn;
+	private String classificationCode;
+	private String censusAddressType;
+	private String censusEstabType;
+//	private Long censusEstabUprn;
```

Comment:
phew found something. Remove m=comments :-)

Reply:
Yes I don't like leaving in commented out code. I think these will be required in the next code change so I just kept them.